src/evaluate/evaluator/question_answering.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2022 The HuggingFace Datasets Authors and the TensorFlow Datasets Authors.
```

Comment:
```suggestion
# Copyright 2022 The HuggingFace Evaluate Authors.
```

Reply:
Thanks a lot, fixed for the other files as well!